test.js

```diff
@@ -0,0 +1,4 @@
+let {CucumberReportPortalHandler} = require('./index.js');
+
+
+console.deb('Vovan')
```

Comment:
It looks like it is 'debug' file. Pls remove it from PR or explain why you need it.

Reply:
removed